src/handler/core.rs

```diff
@@ -194,52 +341,94 @@ where
     /// ```
     pub async fn execute<Cont, ContFut>(
         self,
-        container: Input,
+        input: Input,
```

Comment:
It has turned out to be indeed simpler. The `Input` generic was initially here when DI was optional, but since basically it's required now by almost all combinators, I don't see any reason to leave it generic. Probably one counterargument would be that the `Insert` and `DependencySupplier` traits make it explicit what operations are being invoked by particular functions, but I doubt that somebody really cares.